packages/ui/src/composites/wui-tabs/styles.ts

```diff
@@ -2,7 +2,12 @@ import { css } from 'lit'
 
 export default css`
   :host {
+    --local-tab-width: calc(
+      calc(var(--local-tabs-width) - calc(var(--wui-spacing-3xs) * 2)) / var(--local-tab-count)
```

Comment:
what happens in `local-tab-count = 0` ?  not sure if possible though but just flagging a potential divide by zero error

Reply:
There always should be 1 tab, so I think it is fine